decentraland/unity-renderer
# added fade in/fade out in loading screen

## What does this PR change?

Fix #1767
Adds a fade in and fade out when loading screen appears

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/loading-fade
2. Go from one parcel to another with the events/places in 0,0 or with the map teleport
3. Verify that the loading screen appears with a fade in and disappears with a fade out

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
